pkg/pods/pod.go

```diff
@@ -186,10 +186,6 @@ func (pod *Pod) buildRunitServices(launchables []hoist.Launchable) error {
 		for _, executable := range executables {
 			sbTemplate.AddEntry(executable.Service.Name, executable.Exec)
 		}
```

Comment:
I would like to do this in a separate commit only because I think we need to replace this message with something else. Ideally we provide information to the service healthcheck about the uptime of services and the PIDs they're running as.